doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -775,6 +775,12 @@ This command has been available through the :cmd:`Comments` command,
 e.g. :n:`Comments From … Dependency …`.  The :n:`Comments` form is deprecated
 in Coq 8.16.
 
+.. warn:: File ... found twice in ...
+
+   The file is found in more than once in the top directories
+   associated to the given :n:`@dirpath`. In this case the first occurrence
+   is selected.
```

Comment:
```suggestion
   .. warn:: File ... found twice in ...

      The file is found in more than once in the top directories
      associated with the given :n:`@dirpath`. In this case the first occurrence
      is selected.
```

Please indent by 3 more spaces so it's under the `From .. Extra Dependency` command.

Reply:
Would also be nice to change the wording of the message to "more than once" to make it accurate if there are > 2 uses.  Up to you.